contrib/init/sysvinit-redhat/docker

```diff
@@ -97,6 +99,13 @@ rh_status_q() {
     rh_status >/dev/null 2>&1
 }
 
+
+check_for_cleanup() {
+  if [ -f ${pidfile} ]; then
+    /bin/ps -fp $(cat ${pidfile}) > /dev/null || rm ${pidfile}
+  fi
```

Comment:
ahh sorry about the indentation, I can fix that, There must be something going on with my vim indent not picking up its a sh file. 
To your point it is a little hacky but what I've been seeing is if the docker service dies the pidfile will still be hanging around. So if that happens and the machine reboots. or something is trying to restart the docker service, It won't come back up until someone manually intervenes. Feel free to reject the PR just thought I'd see if it interests anyone.